composer.json

```diff
@@ -11,18 +11,20 @@
   ],
   "require": {
     "php": ">=7.1",
+    "ext-json": "*",
     "ext-pdo": "*",
     "keboola/csv": "^1.1",
     "keboola/db-extractor-logger": "^1.0",
     "keboola/db-extractor-ssh-tunnel": "^1.1",
     "keboola/php-datatypes": "^4.0",
-    "keboola/php-utils": "^2.3",
+    "keboola/php-temp": "^2.0",
+    "keboola/php-utils": "^2.3||^3.0||^4.0",
     "keboola/retry": "^0.4",
     "nette/utils": "^2.4",
     "pimple/pimple": "^3.0",
-    "symfony/config": "^3.0",
-    "symfony/serializer": "^3.1",
-    "symfony/yaml": "^3.0"
+    "symfony/config": "^3.0||^4.3",
+    "symfony/serializer": "^3.1||^4.3",
+    "symfony/yaml": "^3.0||^4.3"
```

Comment:
Je pořád ještě potřeba? Myslel jsem, že jsme se domluvili, že to půjde definitivně do kopru, ne?

Reply:
vím že jste se o tom bavili v jiném PR, každopadně bych to nepletl do tohoto PR ale případně bych to udělal v jiném

Reply:
jj, já jsem právě myslel, že už tam vůbec není v téhle verzi... ale ono to ještě není v10